1_prep.R

```diff
@@ -19,8 +19,8 @@ p1 <- list(
                arrange(site_id)),
 
   # Define mapping variables
-  tar_target(p1_site_ids, p1_lake_cell_tile_xwalk_df$site_id),
-  tar_target(p1_cell_nos, unique(p1_lake_cell_tile_xwalk_df$cell_no)),
+  tar_target(p1_site_ids, p1_lake_cell_tile_xwalk_df %>% pull(site_id)),
+  tar_target(p1_cell_nos, unique(p1_lake_cell_tile_xwalk_df %>% pull(data_cell_no))),
```

Comment:
Note that the change from `$` to `pull()` was added because `$cell_no` returned `NULL` but didn't cause a failure because `cell_no` didn't exist. Using `pull()` throws an error when the column requested doesn't exist, which is the behavior we would rather see here.

Reply:
Writing this out so we have a record of it into the future. No need to do anything based on this comment.